manifests/o/ottercorp/FFXIVQuickLauncherCN/6.2.45-beta2/ottercorp.FFXIVQuickLauncherCN.locale.en-US.yaml

```
@@ -0,0 +1,14 @@
+# Created using wingetcreate 1.1.2.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.2.0.schema.json
+
+PackageIdentifier: ottercorp.FFXIVQuickLauncherCN
+PackageVersion: 6.2.45-beta2
+PackageLocale: en-US
+Publisher: ottercorp
+PackageName: FFXIVQuickLauncherCN

```

```suggestion
PackageName: XIVLauncherCN
```
This must match Control Panel.